libraries/APM_Control/AP_SteerController.h

```diff
@@ -63,6 +68,13 @@ class AP_SteerController {
 	uint32_t _last_t;
 	float _last_out;
 
+    /*
+      variables used by functionality that allows pilot to bias the rate demand when
+      operating in an angle cor lateral acceleration ontrol mode
+    */
+    float _last_rate_bias;      // last received rate bias in deg/sec
+    uint32_t _last_rate_bias_t; // system time _last_rate_bias was updated (msec)
```

Comment:
```suggestion
    uint32_t _last_rate_bias_updated_ms; // system time _last_rate_bias was updated (msec)
```

Not sure about adding `updated`, but  `ms` is kinda standard.  I do note that its being assigned from something ending with `_t`, 'though, so "standard for this file" is enough to dismiss this request.

Reply:
TStandard in this file and the entire APM_Control library so will stick with _t for now but add _updated_ to the name